### PR TITLE
Added new keys to redact

### PR DESCRIPTION
**Pull request by Danielecina**
#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [x] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: lib/custom-logger.js [hunk 1]
```diff
@@ -103,6 +109,7 @@ module.exports = function customLogger(moduleOptions, options) {
   return pinoOptions(moduleOptions, options)
 }
 
+module.exports.defaultRedactionRules = defaultRedactionRules
```
Comment (fredmaggiowski) [inline lib/custom-logger.js:112]: why this export?

Comment (Danielecina) [reply lib/custom-logger.js:112]: to test defaultRedactionRules

File: tests/custom-logger.test.js [hunk 2]
```diff
@@ -46,12 +49,11 @@ test('Test generation custom logger default options', assert => {
     redact: {
```
Comment (davidebianchi) [inline tests/custom-logger.test.js:49]: Here we could now use the exported default redaction rules

File: lib/custom-logger.js (hunk 1 above)
Comment (davidebianchi) [reply lib/custom-logger.js:112]:
```suggestion
module.exports.logDefaultRedactionRules = defaultRedactionRules
```

File: tests/custom-logger.test.js [hunk 3]
```diff
@@ -17,8 +17,11 @@
 'use strict'
 
 const { test } = require('tap')
+const { pick } = require('ramda')
```
Comment (davidebianchi) [inline tests/custom-logger.test.js:20]: We could avoid to use ramda, to remove the added dependency

**Review by mattia-fumo — APPROVED** [review]

**Review by davidebianchi — APPROVED** [review]

**Pull request merged**